amplify/backend/function/BorrowItem/src/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "BorrowItem",
+  "version": "2.0.0",
+  "description": "Lambda function generated by Amplify",
+  "main": ",./ts-output/handlers/api/<BorrowItem>.handler",
```

Comment:
Remove the <> from the string. It should be `./ts-output/handlers/api/BorrowItem.handler`